src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+p {
+  margin: 0;
+}
+
+.card {
+  height: 406px;
+  width: 198px;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    margin: 32px auto 0;
+    height: 134px;
+    width: 160px;
+
+    background-image: url(../images/imac.jpeg);
+    background-size: 100% 100%;
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #060b35;
```

Comment:
Check if the font-size, weight, and line-height match the design requirements. The checklist emphasizes adherence to the design specs.